Mathlib/LinearAlgebra/TensorProduct/Tower.lean

```diff
@@ -0,0 +1,205 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison, Johan Commelin
+-/
+import Mathlib.LinearAlgebra.TensorProduct
+import Mathlib.Algebra.Algebra.Tower
+
+#align_import ring_theory.tensor_product from "leanprover-community/mathlib"@"88fcdc3da43943f5b01925deddaa5bf0c0e85e4e"
+
+/-!
+# The `A`-module structure on `M ⊗[R] N`
+
+When `M` is both an `R`-module and an `A`-module, and `Algebra R A`, then many of the morphisms
+preserve the actions by `A`.
+
+This file provides more general versions of the definitions already in
+`LinearAlgebra/TensorProduct`.
```

Comment:
This is a directory, not a file. The directory contains one file which seems to be about matrices. Should this say something else?

Reply:
Both `LinearAlgebra/TensorProduct/*` and `LinearAlgebra/TensorProduct.lean` exist. This is copied from the old docstring anyway, so I think fixing it is out of scope.

I think we'd like to be able to write `LinearAlgebra.TensorProduct` here (as in the import), but this is written with the hope that doc-gen will know to autolink paths to modules.

Reply:
Aah, maybe `RingTheory/TensorProduct`?

Reply:
Oh apologies, it is a directory *and* a file! I retract my comment.